### PR TITLE
Add a button to stop tracking a website in the options

### LINKED ISSUE #12: Make it possible to remove a website from the list
Both in the settings and with a button in the popup.

### PR DESCRIPTION
**Pull request by elegaanz**
And rewrite the options with Vue (and some extra animations!)

Fix #12

### CODE REVIEW

File: lib/options.js [hunk 1]
```diff
@@ -1,42 +1,36 @@
...
-    }
-    const li = document.createElement('li')
-    li.innerHTML = url
-    list.prepend(li)
-    if (save) {
-      result.config.blacklist.push(url)
-      chrome.storage.local.set(result)
+  },
+  methods: {
+    addWebsite: function (evt) {
```
Comment (KillianKemps) [inline lib/options.js]: Please add tests for at least the `addWebsite` and `remove` functions

**Comment by KillianKemps** [comment]
Thank you for your PR, the animations are nice!
You just need to add some tests and it's okay. You may check at how I wrote tests in #19 to get inspired.
Your PR may help me for my #19 PR as you already converted the options to Vuejs :D

File: lib/options.js [hunk 2]
```diff
@@ -1,42 +1,38 @@
...
+      this.blacklist = result.config.blacklist
+    })
   }
 })
+
+if (module) {
+  console.log('EXPORT\n\n')
+  module.exports = app
+}
+console.log('end of options')
```
Comment (KillianKemps) [inline lib/options.js]: Don't forget to remove the console log once you are finished ;)

Comment (elegaanz) [reply lib/options.js]: Oops! :grin:

**Review by KillianKemps — APPROVED** [review]
Thanks!

**Pull request merged**